sycl/plugins/level_zero/CMakeLists.txt

```diff
@@ -113,6 +113,9 @@ add_sycl_plugin(level_zero
     "${CMAKE_CURRENT_SOURCE_DIR}/usm_allocator.cpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/usm_allocator.hpp"
     "${CMAKE_CURRENT_SOURCE_DIR}/tracing.cpp"
+    "${CMAKE_SOURCE_DIR}/../sycl/plugins/unified_runtime/ur.hpp"
+    "${CMAKE_SOURCE_DIR}/../sycl/plugins/unified_runtime/adapters/level_zero/ur_level_zero.hpp"
+    "${CMAKE_SOURCE_DIR}/../sycl/plugins/unified_runtime/adapters/level_zero/ur_level_zero.cpp"
```

Comment:
Should the new files be included in this PR?

Reply:
absolutely, added